internal/manifest/manifest.go

```diff
@@ -270,6 +263,9 @@ func manifestAddSlices(dbw *jsonwall.DBWriter, slices []*setup.Slice) error {
 }
 
 func manifestAddReport(dbw *jsonwall.DBWriter, report *Report) error {
+	if report == nil {
+		return nil
```

Comment:
Creating a manifest without a report looks like a bug.  What code elsewhere is doing this and why?